evaluation/data/cmc/sorc/regrid_CMCsnow_toGaussianGrid.f90

```diff
@@ -0,0 +1,375 @@
+program data_regrid_gaussian
```

Comment:
seems like a lot of repeated code from the 1D version, maybe we can combine this to one code in the future

Reply:
Let us to do this in future for next iteration.